README.md

```diff
@@ -145,7 +145,7 @@ This project follows the [all-contributors](https://github.com/all-contributors/
 |                                     <a href="https://github.com/harshcasper"><img src="https://avatars1.githubusercontent.com/u/47351025?s=460&u=e6985588320978737a51ac23c8a624005fce5e18&v=4" width=150px height=150px /></a>                                      |
 | :-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
 |                                                                                      **[Harsh Bardhan Mishra](https://www.linkedin.com/in/harsh-bardhan-mishra-b19990173/)**                                                                                       |
-| <a href="https://www.linkedin.com/in/harsh-bardhan-mishra-b19990173/"><img src="https://mpng.subpng.com/20180324/vhe/kisspng-linkedin-computer-icons-logo-social-networking-ser-facebook-5ab6ebfe5f5397.2333748215219374063905.jpg" width="32px" height="32px"></a> |
+| <a href="https://www.linkedin.com/in/harsh-bardhan-mishra-b19990173"><img src="https://mpng.subpng.com/20180324/vhe/kisspng-linkedin-computer-icons-logo-social-networking-ser-facebook-5ab6ebfe5f5397.2333748215219374063905.jpg" width="32px" height="32px"></a> |
```

Comment:
@iamayushdas I have observed that you have removed `/` from the linkedin link. Any specific reasons why this enhancement is needed?
Also according to #805 this section is going to be replaced.